lib/OpenQA/Utils.pm

```diff
@@ -918,8 +918,14 @@ sub compose_job_overview_search_args {
 
     # add simple query params to search args
     for my $arg (qw(distri version flavor build)) {
-        my $param = $controller->param($arg) or next;
-        $search_args{$arg} = $param;
+        my $params = $controller->every_param($arg) or next;
```

Comment:
Not a blocker at all, just a style question. Wouldn't this make more sense written as `next unless my $params = ...;`? Sure a variable is created here, but the fact that there is also control flow logic on this line is much more important for understanding the code. And i think that would be more obvious that way.

Reply:
I didn't know that this is possible. I'll try it the next time.